### ISSUE TITLE
White on white buttons with dark reader

### ISSUE THREAD

**Issue by SuperSandro2000** [daattali/oldschool-github-extension#11]
When using this extensions in combination with dark reader some button are white on white.

![image](https://user-images.githubusercontent.com/7258858/85799168-13216600-b73f-11ea-9aad-0c7f8b502a48.png)

**Comment by daattali** [comment]
I don't know what the dark reader does but this seems to be some css that the dark reader causes.

I can only support the native github experience, I can't make it work with 3rd party decorators

**Comment by SuperSandro2000** [comment]
This is the first time I saw such case and all other extensions that enhance github or any site never had such issues so I don't think it is Dark Readers fault but this extension doing something unexptected.

**Comment by daattali** [comment]
I'm forcing the background colour of the buttons to be light grey. It looks like your dark extension is forcing the font colour to be white. That's what I assume is happening.

**Comment by daattali** [comment]
I tried installing Dark Reader and indeed that's what's happening.

I tried a few things to work around that, but there doesn't seem to be a generic way to make it work across all buttons. 

I'll add this so that it fixes many buttons, but not all. Look at the screenshot below, the items on this page are fine 

![image](https://user-images.githubusercontent.com/952340/85879930-2a953900-b7a9-11ea-938a-0f17224d13d3.png)


But some buttons like the "filter" button here get specific CSS from the dark mode theme that are specific to that button and I cannot override them

![image](https://user-images.githubusercontent.com/952340/85880286-c1fa8c00-b7a9-11ea-9b4b-d232f2dbb346.png)


Also note that the line under the tabs is grey on the sides and black in the middle. The dark reader extension doesn't add any CSS classes to the page's HTML so I can't easily tell whether or not the dark reader is on or off. So this is the best I can do while still staying generic.
